corehq/apps/app_manager/models.py

```diff
@@ -1592,11 +1593,10 @@ def _get_active_actions(self, types):
 
     @memoized
     def get_action_type(self):
-
         if self.actions.close_case.condition.is_active():
             return 'close'
-        elif (self.actions.open_case.condition.is_active() or
-                self.actions.subcases):
+        elif (self.actions.open_case.condition.is_active()
+                or self.actions.subcases):
```

Comment:
It's going to complain about this indent, isn't it?

Reply:
It doesn't, actually.  That's an 8-space indent, which I believe is correct for a wrapped `if` statement like this, though it looks weird because it's an `elif`, which moves the `(` two characters from what we're used to seeing.  Using a 4-space indent in this situation makes the second line look like it's inside the conditional, rather than part of it.